modules/suggestions/reactions.ts

```diff
@@ -36,11 +37,56 @@ export async function addToDatabase(thread: AnyThreadChannel): Promise<void> {
 	const defaultEmoji = config.channels.suggestions?.defaultReactionEmoji;
 	const message = await thread.fetchStarterMessage().catch(() => void 0);
 	const count = (defaultEmoji?.id && message?.reactions.resolve(defaultEmoji.id)?.count) || 0;
+	const suggestionData = getSuggestionData(thread);
+	suggestionsDatabase.data = [...suggestionsDatabase.data, { ...suggestionData, count }];
+	const data = suggestionsDatabase.data as {
+		answer: typeof suggestionAnswers[number];
+		author: Snowflake;
+		count: number;
+		id: Snowflake;
+		title: number | string;
+	}[];
 
-	suggestionsDatabase.data = [
-		...suggestionsDatabase.data,
-		{ ...getSuggestionData(thread), count },
-	];
+	const dupes = await findDuplicates(suggestionData, data);
+	if (dupes.length == 0) return;
+	const links = dupes
+		.toSorted((a, b) => a.score - b.score)
+		.toReversed()
+		.map((dupe) => {
+			return `<#${dupe.id}> ${dupe.answer}`;
+		});
+	thread.send(`
+	## Possible dupes found:\n${links.join("\n")}
+	`);
+}
+
+export async function findDuplicates(
+	newSuggestion: {
+		id: string;
+		title: string | number;
+	},
+	database: {
+		id: Snowflake;
+		title: number | string;
+		answer: string;
+	}[],
+) {
+	let possibleDupes: {
+		id: Snowflake;
+		title: number | string;
+		answer: string;
+		score: number;
+	}[] = [];
+	for (const dbSuggestion of database) {
+		if (dbSuggestion.id == newSuggestion.id) continue;
+		const similarityThreshold = 0.5;
+		const similarity = stringSimilarity(`${newSuggestion.title}`, `${dbSuggestion.title}`);
```

Comment:
string-similarity-js isn't even installed. Look at how Scradd Chat does this using didyoumean2.